bgpd/bgp_evpn_mh.h

```diff
@@ -335,6 +335,7 @@ extern int bgp_evpn_es_route_install_uninstall(struct bgp *bgp,
 		struct bgp_evpn_es *es, afi_t afi, safi_t safi,
 		struct prefix_evpn *evp, struct bgp_path_info *pi,
 		int install);
+extern void update_type1_routes_for_vni(struct bgp *bgp, struct bgpevpn *vpn);
```

Comment:
I know I still have some work to do to rid these functions of `vni` but could we make this `evi` instead?